src/styles/main.scss

```diff
@@ -0,0 +1,75 @@
+$item-bg-color: #eee;
+$calendar-item-size: 100px;
+$calendar-gap: 1px;
+$calendar-item-width: calc(#{$calendar-item-size} + #{$calendar-gap});
+$week: (
+  'mon': 0,
+  'tue': 1,
+  'wed': 2,
+  'thu': 3,
+  'fri': 4,
+  'sat': 5,
+  'sun': 6,
+);
+
+* {
+  margin: 0;
+  padding: 0;
+  font-family: Arial, sans-serif;
+  font-size: 30px;
+}
+
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  padding: 10px;
+  gap: $calendar-gap;
+  width: calc($calendar-item-width * 7);
+
+  &__item {
+    box-sizing: border-box;
+    background-color: $item-bg-color;
+    height: $calendar-item-size;
+    width: $calendar-item-size;
+    border: 1px solid black;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    &:hover {
+      cursor: pointer;
+      background-color: #ffbfcb;
+      transform: translateY(-20px);
+      transition-duration: 0.5s;
```

Comment:
The `transition-duration` should not be placed within the `:hover` selector. It's better to define `transition` property on the element itself (`.calendar__item`), not on the hover state, so that the transition effect is smooth both when the mouse enters and leaves the element.